Amazon.QLDB.Driver/retry/IAsyncRetryHandler.cs

```diff
@@ -20,11 +20,11 @@ namespace Amazon.QLDB.Driver
     internal interface IAsyncRetryHandler
     {
         Task<T> RetriableExecute<T>(
```

Comment:
This seems odd to me but we can revisit this once more implementation is completed.